src/zimscraperlib/zim.py

```diff
@@ -100,11 +100,26 @@ def to_zimwriterfs_args(
         return arg_list
 
 
-def make_zim_file(build_dir, output_dir, zim_fname, zim_info):
-    """ runs zimwriterfs """
+def make_zim_file(build_dir, output_dir, zim_fname, zim_info, **kwargs):
+    """ runs zimwriterfs 
+    
+    The function takes in the following arguments
+        build_dir: The directory to convert into a ZIM file
+        output_dir: The directory where the ZIM file would be saved
+        zim_fname: The name of the ZIM file
+        zim_info: An object of class ZimInfo
+    
+    Also supports the following keyword arguments that are passed to to_zimwriterfs_args()
```

Comment:
Good to mention is supports `to_zimwriterfs_args()` but that's enough IMO. That's not the place to describe those. We could do that in the other method but it's advanced feautres anyway.